opendatadiscovery/odd-platform
# Improve Slack notification message and fix leakages

**What changes did you make?** (Give an overview)
This PR contain improvements for Slack/Webhook notifications. Several bugs and leakages were fixed. Slack message formatting has been improved.

**How to test**
The best way would be:
1. Create Slack workspace and create an incoming webhook.
2. Configure the platform to send notifications via Slack
3. Inject complex data graph and emulate several Alert types using Ingestion API (remove an existing column, inject failed job run/test run)
4. Assert that Slack messages appear and they are providing correct data

closes #742
